### ISSUE TITLE
Kontrola dateIssued u NDK Čísla a NDK Přílohy Ročníku/Čísla

### ISSUE THREAD

**Issue by SykoraLukas** [proarc/proarc#1715]
API podpora pro https://github.com/proarc/proarc-client/issues/378 a https://github.com/proarc/proarc-client/issues/399

Týká se modelů NDK číslo periodika, NDK příloha periodika.

Zadání viz issue na klientovi - kontrola formátu a relevantnosti hodnot.

**Comment by ZdenkaSera** [comment]
@SykoraLukas Lukáši, tento vývoj je ve vydané verzi 4.2.2 a obě související issues na klientovi jsou označena k testu. Předpokládám, že z hlediska funkčnosti na klientovi k tomu není zapotřebí vývoj, hlášení z api se promítají do správy procesů, takže lze otestovat lokálně na verzi 4.2.2 s klientem 2.3.1? (Vývoj na klientovi se bude týkat vylepšeného zobrazování chyb v úložišti, že?) Díky.

**Comment by SykoraLukas** [comment]
Ano, k tomuto nebylo zapotřebí vývoj na klientovi. Při uložení formuláře se kontroluje formát a validita dateIssued.

Při spuštění validace a exportu se kontroluje obojí + odpovídající rok v číslu/příloze k ročníku.

Na klientovi se tedy již neplánuje k odkazovaným 2 issue žádný vývoj. Jediné, co se plánuje je lepší zobrazení mezi chyby validace u daného objektu v obrazovce hledat (aby uživatel nemusel koukat do správy procesů, ale viděl to přímo v obrazovce hledat.
